python/message.c

```diff
@@ -935,10 +935,19 @@ static PyObject* PyUpb_CMessage_IsInitializedAppendErrors(PyObject* _self,
                                                           PyObject* errors) {
   PyObject* list = PyUpb_CMessage_FindInitializationErrors(_self, NULL);
   if (!list) return NULL;
-  bool ret = PyList_Size(list) == 0;
-  if (!ret) PyObject_CallMethod(errors, "extend", "O", list);
+  bool ok = PyList_Size(list) == 0;
+  PyObject* ret = NULL;
+  PyObject* result = NULL;
+  if (!ok) {
+    result = PyObject_CallMethod(errors, "extend", "O", list);
```

Comment:
having objects named both `ret` and `result` is a bit confusing.  Please pick a more clear name for `result`

Reply:
Renamed to `extend_result`.